benchmark/scenarios/rate_measurer.py

```diff
@@ -0,0 +1,70 @@
+# Copyright 2015 ClusterHQ Inc.  See LICENSE file for details.
```

Comment:
New class with all the common functionality that was duplicated in the read and write scenarios